apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-save-click.js

```diff
@@ -19,7 +19,12 @@ export const wpcomBlockEditorSaveClick = () => ( {
 		let actionType = 'publish';
 
 		if ( isSiteEditor ) {
-			actionType = 'save';
+			const isPreviewingBlockTheme = window.location.search.includes( 'wp_theme_preview' );
```

Comment:
It might be better to use [isPreviewingTheme](https://github.com/WordPress/gutenberg/blob/12f305ece929393fc246544b03f7881e8a0923a1/packages/edit-site/src/utils/is-previewing-theme.js#L6) if possible. Otherwise, we can simpy copy & paste the implementation to here as using `includes` might not be very accurate 😅

Reply:
I had a go for another way; https://github.com/Automattic/wp-calypso/pull/81000/commits/705c9324a03860533b180d3d4bd05cc1d4cd7313